jsonnet/kube-prometheus/addons/podsecuritypolicies.libsonnet

```diff
@@ -0,0 +1,256 @@
+local restrictedPodSecurityPolicy = {
+  apiVersion: 'policy/v1beta1',
+  kind: 'PodSecurityPolicy',
+  metadata: {
+    name: 'restricted',
```

Comment:
There are many providers out there that manage a `restricted` PSP for you. I would suggest that we rename this to `kube-prometheus-restricted`.